www/src/libs/math.js

```diff
@@ -317,6 +317,7 @@ var _mod = {
        var x1=float_check(x);
        var y1=float_check(y);
        return float(Math.sqrt(x1*x1 + y1*y1))},
+    inf: float(Infinity),
```

Comment:
The MDN [reference for Infinity](https://developer.mozilla.org/cs/docs/Web/JavaScript/Reference/Global_Objects/Infinity) says that *initially* `Infinity` is `Number.POSITIVE_INFINITY`. However (see [discussion](http://stackoverflow.com/questions/8522400/infinity-vs-number-positive-infinity) on StackOverflow), in previous versions of the ECMAScript spec the `Infinity` property *was overwritable* (e.g. you could write `Infinity=10`). As of ES5 it is readonly. For backwards compatibility it might still be safer to explicitly define `inf` as`Number.POSITIVE_INFINITY`.